app/views/posts/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2>Members are saying:</h2>
+
+<ul class="posts">
+  <% @posts.each do |post| %>
+    <li>
+    <%= link_to post.title, post_path(post) %><br />
```

Comment:
Why `post_path`? 
We got an error when clicking on the link
![Screenshot_2020-05-30_23-30-46](https://user-images.githubusercontent.com/8848104/83340244-9f21a880-a2cd-11ea-9250-00333bfac70b.png)

It's because the `post_path` route doesn't exists.
It's better remove it as a link and just write the post's title and description